docs/tutorials/subscription-platform.md

```diff
@@ -485,63 +496,20 @@ Before you begin, make sure you have App Store API credentials set up in the aut
-4. Link the GCP project's Firestore instance to the auth server in `fxa-auth-server/config/secrets.json`.
-      - Add the below configuration key/value pairs under `subscriptions`.
-      - Replace the value for `keyFilename` with the absolute path to the keyfile created in step 3.
-      - Replace the value for `projectId` with the project ID for the GCP project created in step 3.
-
-```js
-{
-    // ...
-    "subscriptions": {
-        // ...
-    },
-    "authFirestore": {
-        "keyFilename": "",
-        "projectId": ""
-    }
-}
-```
-5. Restart the auth server to point to the new Firestore instance
-    - We use a Firestore emulator by default for local development, but for IAP, we want to point our local FxA to the newly created Firestore instance. We can do this by unsetting the `FIRESTORE_EMULATOR_HOST` environment variable and restarting the auth server.
-
-```
-FIRESTORE_EMULATOR_HOST='' pm2 restart auth --update-env
-```
-
-### Put the subscription in Firestore
-
-#### Google IAP
-
-1. In the GCP project, create a new Firestore collection named `${authFirestore.prefix}iap-play-purchases`. You can find the default prefix in `fxa-auth-server/config/index.ts`.
-2. Copy a test Google IAP subscription purchase object from the `fxa-auth-stage-iap-play-purchases` collection in [FxA stage Firestore](https://console.cloud.google.com/firestore/data/fxa-auth-stage-iap-play-purchases?project=moz-fx-fxa-nonprod-375e) as a new document in the new collection.
-3. Manually edit `expiryTimeMillis` to be a date in the distant future.
-4. Create a new FxA user locally and set their user id as the `userId` value.
-
-See [Verify it worked](#verify-it-worked) to confirm everything works correctly.
-
-#### Apple IAP
-
-1. In the GCP project, create a new Firestore collection named `${authFirestore.prefix}iap-app-store-purchases`. You can find the default prefix in `fxa-auth-server/config/index.ts`.
-2. After [QA-1391](https://mozilla-hub.atlassian.net/browse/QA-1391) is resolved, copy a test Apple IAP subscription purchase object from the `fxa-auth-stage-iap-app-store-purchases` collection in [FxA stage Firestore](https://console.cloud.google.com/firestore/data/fxa-auth-stage-iap-play-purchases?project=moz-fx-fxa-nonprod-375e) into the new collection.
-3. Manually edit `expiresDate` to be a date in the distant future and set `status` to `1`.
-4. Create a new FxA user locally and set their user id as the `userId` value.
-
-See [Verify it worked](#verify-it-worked) to confirm everything works correctly.
+    - Enable the IAP integration. See [Google IAP Integration](#google-iap-integration) for Google IAP and [Apple IAP Integration](#apple-iap-integration) for Apple IAP.
+    - Note: For most development, setting `enabled` for both relevant configs is enough without setting up credentials.
+2. Setup a price for IAP
+    - Note: This step is optional, since you can use `examplesku` for a pre-configured Stripe price.
```

Comment:
Since the prices/metadata in the `SUB_PLAT_DEV` Stripe account dashboard can change at any time, I wouldn't assume whatever plan currently has this configuration (`examplesku`) will be available. The important thing for people to know is what to look for on any existing plans and what to add if they don't find one.

Also, at some point in the near future, we will be migrating our product/plan configs to Firestore and stop using the Stripe dashboard altogether. Might be good to make these docs future proof by removing references to creating/editing prices in Stripe and just refer to plan/product configs whereever they happen to reside.

Reply:
> can change at any time

That's fair, though I did like having this here since it cuts a step.

>  migrating our product/plan configs to Firestore

Yeah, definitely true that this becomes an issue. I wanted to be specific enough to help guide someone who has no idea how to set this type of thing up (new to the team). I'll adjust to reference both instead.